lib/rbs_rails/active_record.rb

```diff
@@ -66,21 +66,45 @@ class #{klass.name}::ActiveRecord_Associations_CollectionProxy < ActiveRecord::A
 
       private def has_many
         klass.reflect_on_all_associations(:has_many).map do |a|
-          "def #{a.name}: () -> #{a.klass.name}::ActiveRecord_Associations_CollectionProxy"
+          singular_name = a.name.to_s.singularize
+          type = a.klass.name
+          collection_type = "#{type}::ActiveRecord_Associations_CollectionProxy"
+          <<~RUBY.chomp
+            def #{a.name}: () -> #{collection_type}
+            def #{a.name}=: (#{collection_type} | Array[#{type}]) -> (#{collection_type} | Array[#{type}])
+            def #{singular_name}_ids: () -> Array[Integer]
```

Comment:
I guess id can be non-Integer, but I just skipped supporting it. It's future work.